src/__myproject__/_export/clickhouse_export.py

```diff
@@ -119,3 +119,14 @@ def rename_tables(table_names: dict, logger: Logger):
     execute_clickhouse_query(f"RENAME TABLE {table_names['temp']} TO {table_names['main']}")
 
     logger.info(f"Export successful, table saved as {table_names['main']}.")
+
+# COMMAND ----------
+
+@dp.notebook_function(features_to_export_with_conversions, get_table_names)
+def compare_counts(df: DataFrame, table_names: dict, logger: Logger):
+    fs_count = df.count()
+    clickhouse_count = execute_clickhouse_query(f"SELECT COUNT(*) from {table_names['main']}")[0]["count()"]
+    if int(fs_count) == int(clickhouse_count):
```

Comment:
they are not ints?

Reply:
One of them wasn't.

Reply:
Removed fs_count casting and moved clickhouse_count casting up.